docs/sources/clients/promtail/configuration.md

```diff
@@ -930,6 +933,55 @@ labels:
 [use_incoming_timestamp: <bool> | default = false]
 ```
 
+### GCP Log
+
+The `gcplog` block configures Promtail receive GCP logs. It has two possible strategies for doing so, depending on how the 
+`subscription_type` attribute is configured:
+1. **Pull**: Using GCP PubSub [pull subscriptions](https://cloud.google.com/pubsub/docs/pull). Promtail will consume log messages directly from the configured GCP PubSub topic.
+2. **Push**: Using GCP PubSub [push subscriptions](https://cloud.google.com/pubsub/docs/push). Promtail will expose an HTTP server, to which GCP will deliver logs.
+
+When picking the `push` subscription type, keep in mind the following:
+- The `server` configuration is the same as [server](#server), since Promtail exposes an HTTP server for target that requires so.
+- An endpoint at `POST /gcp/api/v1/push`, which expects requests from GCP PubSub message delivery system.
```

Comment:
```suggestion
- The `server` configuration is the same as [server](#server), since Promtail exposes an HTTP server for the target that requires it to be the same.
- An endpoint at `POST /gcp/api/v1/push`, which expects requests from a GCP Pub/Sub message delivery system.
```